Deeploy/EngineExtension/NetworkDeployers/EngineColoringDeployer.py

```
@@ -62,8 +62,11 @@ def _initEngineColoringDeployer(self, engineMapperCls: Type[EngineMapper]):
 
     def lower(self, graph: gs.Graph) -> gs.Graph:
         graph = super().lower(graph)
-        uncoloredNodes = [node.name for node in graph.nodes if "engine" not in node.attrs]
-        assert len(uncoloredNodes) == 0, f"Missing engine color for nodes {uncoloredNodes}"
+        uncoloredNodes = [node for node in graph.nodes if "engine" not in node.attrs]
+        uncoloredOperations = set(node.op for node in uncoloredNodes)
+        assert len(
```

Ruff (0.12.2)

Use of `assert` detected

(S101)